### PR TITLE
Release/1.2.21

### PR DESCRIPTION
**Pull request by davyskiba**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+
+- Presence.endSync throws NullPointerException when processing a message [\#853](https://github.com/ably/ably-java/issues/853)
+
+**Closed issues:**
+
+- handling of channel options in InternalChannels.get is not thread safe [\#663](https://github.com/ably/ably-java/issues/663)
+- Remove hardcoded name from Maven Gradle files [\#565](https://github.com/ably/ably-java/issues/565)
+- Android CI fails due to unaccepted licenses [\#554](https://github.com/ably/ably-java/issues/554)
+- AsyncHttpScheduler.dispose\(\) is never used [\#523](https://github.com/ably/ably-java/issues/523)
+- More Encapsulation Needed [\#508](https://github.com/ably/ably-java/issues/508)
```
Comment (ikbalkaya) [inline CHANGELOG.md:16]: I would remove all of these items as these are internal issues

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+**Merged pull requests:**
+
+- added null check to prevent NullPointerExceptions [\#873](https://github.com/ably/ably-java/pull/873) ([davyskiba](https://github.com/davyskiba))
+- Stop hiding flakey test failures [\#861](https://github.com/ably/ably-java/pull/861) ([QuintinWillison](https://github.com/QuintinWillison))
```
Comment (ikbalkaya) [inline CHANGELOG.md:21]: And this particular one ('stop hiding flakey....')

**Review by ikbalkaya — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (davyskiba) [reply CHANGELOG.md]: de6f2563941e1ea4983fc2aa34502f016b8c99c0

File: CHANGELOG.md (hunk 2 above)
Comment (davyskiba) [reply CHANGELOG.md]: de6f2563941e1ea4983fc2aa34502f016b8c99c0

**Review by ikbalkaya — APPROVED** [review]

**Pull request merged**